Hit api

peterdgreat opened the pull request:
-  I created a new game by using the API.
- I Implemented the "Refresh" button 
- I  Implemented the form "Submit" button (sending data to the API).
- I used async and await JavaScript features to consume the API.

peterdgreat committed:
Game.js created

- new Id generated from adding new game
- exported to index js

peterdgreat committed:
Id generated on clicking refresh button

peterdgreat committed:
Update index.html

-  commented out the written html list items

peterdgreat committed:
Score and Name From User input

peterdgreat committed:
Local storage setup

- On reload and refresh, score id is stored in local storage

peterdgreat committed:
Local Storage removed and Data gotten from the api service

peterdgreat committed:
Refresh

- A new Game created on refresh
- local storage added to preserve id

BrianSammit commented on src/index.js:
```diff
@@ -1 +1,38 @@
+  li.appendChild(score);
+  list.appendChild(li);
+};
+refresh.addEventListener('click', () => {
+  list.innerHTML = '';
+  Game.refresh();
+});
```
- [ ] It would be better if when you click on the refresh page the data doesn't disappear. The job of the refresh button is to receiving data from the API and parsing the JSON. kindly fix this minor issue. 

![Capture](https://user-images.githubusercontent.com/64173189/136498363-980938ef-7a09-4da9-be11-a9e1a4b4ad00.JPG)

🛠️ 😄

peterdgreat replied:
Wow, thanks will fix that immediately, this means refresh is the same work as when windows load

BrianSammit reviewed (changes requested):
# CHANGES REQUIRED 🔴 🛑
Hello @peterdgreat, great job you are doing so far in this project. 💯
TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 

However, there are a few changes to make to move to the next project 🛠️ . Keep working hard 💪 💻 

## Required Changes ♻️
See my inline comment to know what changes are needed. 👇 👀 

Please make these required changes and send it again for another review 👍 💪
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
# HAPPY CODING!!! 😄 

![cat](https://user-images.githubusercontent.com/64173189/131593916-14c184cb-b5af-40f7-8fba-779155f154cb.gif)


**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

--------

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

peterdgreat committed:
Refresh event  listener modified

- On clicking button refreshes, it calls out the window reload which automatically loads up the api

dasileker reviewed (approved):
### **Hi @peterdgreat,** 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 

![Approved](https://user-images.githubusercontent.com/61562626/136536008-9b82a9fb-1e4c-4392-a598-e2d2e545dd1c.gif)



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
